Dockerfile

```diff
@@ -38,4 +38,3 @@ WORKDIR core/
 # Command to run any command provided by the COMMAND env variable.
 # Use the command listed at the top to run the voting script repeatedly in a 60 second loop.
 ENTRYPOINT ["/bin/bash", "scripts/runCommand.sh"]
-CMD ["--network=test"]
```

Comment:
Just out of curiosity -- what was this doing?

Reply:
if you combined an `ENTRYPOINT` and a `CMD` it will call the CMD after the ENTRYPOINT. However, my research showed that this is bad practice as it can sometimes do inconsistent things when combined together. I thought it best to remove this because we specify the network in the `COMMAND`.

Reply:
SGTM.